homework1/src/RemoveDuplicates.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+
+public class RemoveDuplicates {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 2, 3, 3, 10, 11, 20};
+        Arrays.sort(numbers, Collections.reverseOrder());
+
+        System.out.print(Arrays.toString(numbers));
+
+        LinkedHashSet<Integer> linkedHashSet =  new LinkedHashSet<>(Arrays.asList(numbers));
```

Comment:
Consider using Set in the first part:
`Set<Integer> linkedHashSet =  new LinkedHashSet<>(Arrays.asList(numbers));
`

Reply:
it makes our collection more flexible

Reply:
Fixed in commit: f1536a4